### PR TITLE
Rector vs CS

### PR DESCRIPTION
**Pull request by slunak**
Looks like rector and cs conflict... see commits @OskarStark

### CODE REVIEW

**Comment by OskarStark** [comment]
Will have a look tomorrow

**Review by Vitexus — APPROVED** [review]
image
there is nothing here to break

**Comment by OskarStark** [comment]
CleanShot 2024-09-02 at 08 04 15@2x

This is what we want, right?

**Comment by slunak** [comment]
> This is what we want, right?
I dont mind either, depending what is the standard in PHP, up to you to decide basically :)

**Comment by OskarStark** [comment]
Yes, so this is what we want

File: examples/CompleteNotificationExample.php [hunk 1]
```diff
@@ -43,7 +44,7 @@ public function completeNotification(): void
         $message->setUrl('https://www.example.com');
         $message->setUrlTitle('Example URL');
         $message->setisHtml(false);
-        $message->setTimestamp(new \DateTime('now'));
```
Comment (slunak) [inline examples/CompleteNotificationExample.php]: So in this case rector should not do this, right?

**Pull request closed without merging**